controllers/argocdmetrics/argocd_metrics_controller.go

```diff
@@ -1,8 +1,25 @@
+/*
```

Comment:
The `Reconcile` function in this file still has the old signature `Reconcile(reconcile.Request) (reconcile.Result, error)`.  Shouldn't it have the new signature `Reconcile(context.Context, reconcile.Request) (reconcile.Result, error)` that was generated by `operator-sdk create api ...`?

Reply:
Signatures will be updated in a followup PR along with the latest argocd-operator